src/leveldb/util/env_posix.cc

```diff
@@ -335,7 +335,7 @@ class PosixWritableFile final : public WritableFile {
     return status;
   }
 
-  Status WriteUnbuffered(const char* data, size_t size) {
+  Status WriteUnbuffered(const char* data, size_t size) const {
```

Comment:
You are touching files in https://github.com/bitcoin-core/leveldb-subtree; changes to it need to be made there or upstream and not in bitcoin/bitcoin. See the CI linter failure: "FAIL: subtree directory was touched without subtree merge" (`test/lint/git-subtree-check.sh`)

Reply:
Thanks, I was wondering what this meant.
I removed these changes.